### ISSUE TITLE
Question about differed debit cards

### ISSUE THREAD

**Issue by a491765** [firefly-iii/firefly-iii#2764]
I am running Firefly III version 4.8.1.4

**Description**

I wonder how differed debit cards works in firefly_iii.
I.E in my case, i have a differed debit card which works from the 20 of a month to the 19 of the other month (I.E 2019/09/20 > 2019/10/19) all purchase are globalized and there is only one debit on my account on the 30rd of the month (I.E 2019/10/30).

How can i said to firefly that my purchase are from the 20 to 20 and not from 1 to 30 ?
How to make a budget (in example for groceries or lunch) which begin the 20 and finish the 19 of the month late.

Thanks for you help.

**Comment by JC5** [comment]
You can't. Firefly III works per month, and you can't customize the range. 

Are you paying the debit from another asset account that's also in Firefly III?

**Comment by a491765** [comment]
Actually i created an account call DIFFERED DEBIT CARD which work in negative, and i make a transfert from the account with which the credit card works every 29 of the month to make it back to 0.

**Comment by JC5** [comment]
That should work, it's similar to what I do for credit cards.

**Comment by a491765** [comment]
Ok so it works fine for the differed but its not possible to do budget
Thanks.

**Comment by JC5** [comment]
For the total amount, not really. But the separate transactions can be budgeted.

**Comment by a491765** [comment]
i already do separate transaction with categories. But i mean real budget I.E 100 € for launch between 09/20 and 10/19

**Comment by JC5** [comment]
It's purely cosmetic at the moment. 

The fact that you have to pay the card on the 20th of each month doesn't mean you **have** to budget accordingly. That's entirely up to you. Start separating the monthly payback from what's actually happening, OK? Paying off the card is cosmetic.

You spend X amount during the month on the card. The fact that you have to pay it back the 20th has no influence *at all* on your budget. The money is spent already anyway. It's just moving money around. The transfer to pay the card shouldn't influence your budget.

**Comment by a491765** [comment]
I think i understand what you meant. The only problem is that my bank do 2 sub-accounts to separate operations of each month. so it's a bit hard to follow payment of the month and payments of the next month who already begin those days.

**Comment by JC5** [comment]
That doesn't make it easier, no :(

**Comment by a491765** [comment]
Anyway, i will find how to figure out with them.
Thanks for help and your time.
